libs/core/src/lib/calendar/calendar.component.scss

```diff
@@ -1,4 +1,5 @@
 @import '~fundamental-styles/dist/calendar';
+@import '~fundamental-styles/dist/button';
```

Comment:
Not sure it's a good idea to load button styles into calendar styles. It make sense only if calendar component doesn't use fd-button component itself. Otherwhise it will create some overrides to the styles

Reply:
Yea, we're actually applying button classes to the calendar elements. 

It's a good idea to apply only certain button mixins to that elements instead of importing whole component styles, but to be honest it will take one more refactoring iteration for the button component and i'm not sure that we have time for this now. What do you think about refactoring it later after we finished with the initial horizon support phase?